docs/source/user/core-structures.rst

```diff
@@ -0,0 +1,28 @@
+Core Structures
+===============
+
+.. contents:: |toctitle|
+    :local:
+
+Overview
+--------
+
+OpenDP is about creating computations with specific privacy characteristics. These computations are modeled with two core structures in OpenDP: Transformations and Measurements. These structures are in all OpenDP programs, regardless of the underlying algorithm or definition of privacy. By modeling computations in this abstract way, we're able to combine them in flexible arrangements, and reason about the resulting programs.
+
+Measurement
+-----------
+
+A Measurement is a randomized mapping from datasets to outputs of an arbitrary type.
+
+Transformation
+--------------
+
+A Transformation is a (deterministic) mapping from datasets to datasets.
+
+Constructors and Functions
+--------------------------
+
+In OpenDP, Measurements and Transformations are created by calling constructor functions. Because Measurements and Transformations are themselves like functions (they can be invoked on an input and return an output), you can think of constructors as higher-order functions: You call them to produce another function, that you will then feed some data.
+
+Examples
+--------
```

Comment:
Yes, examples here would be good. It could also be helpful to have an example or a measurement and a transformation above, where they are defined.

The measurement definition itself uses terms that may not be familiar. What kind of outputs? What kind of types?